crates/api_models/src/analytics/disputes.rs

```diff
@@ -64,3 +69,75 @@ impl From<DisputeMetrics> for NameDescription {
         }
     }
 }
+
+use crate::enums::DisputeStage;
```

Comment:
move this import to start of file,
would prefer having these imports in the disputes file maybe..